### DISCUSSION TITLE
Filter files in Source Control bar

### DISCUSSION THREAD

**Discussion by evenkov** [gitkraken/vscode-gitlens#3749] [Ideas]
Hi,
It would be nice if there is option to filtered files in Source Control bar by some criteria with `wildcard` or `regex`. When I make changes in my branch I always make all changes in files and then group them in different commits. It is easy to filter files by some criteria and then only visible ones to `stage`, `unstage`, `stash`, `pop stash`, `discard` and so on. Before I discovered GitLens I often use GitExtension only to use this useful functionality on big number of files than was changed and ready to commit or discard

**Comment by eamodio** [comment]
Unfortunately, GitLens doesn't have the ability to control the `Source Control` view (other than add additional commands).

But this is something we will keep in mind for our own views.
